invitation-frontend/src/component/Visual3.tsx

```diff
@@ -1,5 +1,723 @@
+
+  const optionStyle = (theme: any, isClicked: boolean) => ({
+    root: {
+      flex: 1,
+      marginRight: 10,
+      marginTop: '3px',
+      '&:not([data-disabled])': theme.fn.hover({
+        backgroundColor: theme.fn.darken('#B39884', 0.05)
+      }),
+      backgroundColor: isClicked ? 'rgb(180, 152, 133)' : 'rgb(204, 204, 204)'
+    }
+  });
+
+  const handleOptionClick = (option: 'firstOption' | 'secondOption') => {
+    if (option === 'firstOption') {
+      setIsFirstOptionClicked(true);
+      setIsSecondOptionClicked(false);
+    } else if (option === 'secondOption') {
+      setIsSecondOptionClicked(true);
+      setIsFirstOptionClicked(false);
+    }
+  };
+
+  const handleFormSubmit = (e: React.SyntheticEvent) => {
+    e.preventDefault();
+  };
+
+  const imgs: string[] = [
+    require('../assets/img/visual3/moment_pics/1.png'),
+    require('../assets/img/visual3/moment_pics/2.png'),
+    require('../assets/img/visual3/moment_pics/3.png'),
+    require('../assets/img/visual3/moment_pics/4.png'),
+    require('../assets/img/visual3/moment_pics/5.png')
+  ];
+
+  const slides: JSX.Element[] = imgs.map((img) => {
+    return (
+      <div key={`side-${img}`}>
+        <img src={img} alt={`Slide $(item)`} />
+      </div>
+    );
+  });
+
+  return (
+    <>
+      <MantineProvider
+        theme={{
+          globalStyles: (_theme) => ({
+            '*, *::before, *::after': {
+              boxSizing: 'border-box',
+              fontFamily: 'Kopub Batang!important'
+            },
+            body: {
+              color: '#222'
+            },
+            a: {
+              color: 'inherit!important',
+              textDecoration: 'none'
+            },
+            figure: {
+              margin: 0
+            },
+            ul: {
+              listStyle: 'none',
+              padding: 0
+            },
+            dl: {
+              margin: 0
+            },
+            dd: {
+              margin: 0
+            }
+          })
+        }}
+      >
+        <main
+          className={`${classes.invitationWrap} ${classes.invitationVisual}`}
+        >
+          <Container
+            className={classes.newVisual}
+            fluid
+            px={0}
+            size={RESPONSIVE_MOBILE ? 550 : 450}
+          >
+            <div className={classes.weddingMainDate}>
+              <small className={classes.weddingMainDateSmall}>{year}</small>
+              <strong className={classes.weddingMainDateStrong}>
+                {monthNum}/{day}
+              </strong>
+            </div>
+            <div className={classes.weddingDetails}>
+              <p className={classes.weddingCharacter}>
+                {bride} & {groom}
+              </p>
+              <p>@ {location}</p>
+              <p>
+                {monthNum}.{day}.{year} / {time}
+              </p>
+            </div>
+          </Container>
+          {/* TODO: Need to adjust text inside image */}
```

Comment:
I see this TODO comment now. Do you think we can have those changes in this PR?

Reply:
As we discussed earlier, if arranging texts is difficult, we can place text in the middle. 
However, the issue was easily resolved with some style adjustments. 

I forgot to delete the comment before this PR. I'll delete it.